### PR TITLE
Update members

### PR DESCRIPTION
**Pull request by antoinegag**
This PR updates the members and staff of MMD.

I still need confirmation for some images so don't pull just yet.

### CODE REVIEW

File: data/staff.json
Comment (Darkhax) [inline data/staff.json]: Jared isn't a dev server admin. He is a bot maintainer. Also Diluv.

File: data/staff.json
Comment (Darkhax) [inline data/staff.json]: I actually don't work on any of the MMD mods directly. (unless you count all of my mods, since they're technically diluv mods as well). I would prefer to be listed as `MMD  Admin | Diluv though.`

**Review by Darkhax — CHANGES_REQUESTED** [review]

File: data/staff.json
Comment (antoinegag) [reply data/staff.json]: I had Jared for Diluv in my head too, no idea how that ended up like this, fixed.

**Comment by Darkhax** [comment]
Additionally, Jared's Avatar can be obtained [here](https://cdn.discordapp.com/avatars/79179147875721216/6a573212406b5dd7e9665519eda645f9.webp) although he may want to provide a different file in the future.

**Review by Darkhax — APPROVED** [review]

**Comment by antoinegag** [comment]
I have asked him for one, waiting on an answer.

**Comment by Darkhax** [comment]
The one I linked is acceptable for now.

**Review by jriwanek — APPROVED** [review]

**Review by Dizzy611 — APPROVED** [review]
Looks good to me

**Pull request merged**
